popde/density_estimate.py

```diff
@@ -0,0 +1,91 @@
+from awkde import GaussianKDE 
+
+
+class  DensityEstimator(object):
```

Comment:
In particular, the development plan says that we should also have a class for a general, simple density estimator which is _not_ adaptive or optimized etc.
So that will be another class in addition to the one you wrote here, and the name DensityEstimator might be suitable for that general, simple class, but it is not suitable for the one you have written.